sfcasts/install.md

```diff
@@ -1,37 +1,38 @@
 # Install
 
-Hey friends! We are in for a treat with this tutorial! It's EasyAdmin, my
-favorite admin generated for Symfony. It just gives you so many features out of
-the box and it looks great. This shouldn't really be a surprise because its creator is
-the always-impressive Javier Aguila. So let's have some fun and learn how to bend
-EasyAdmin to our will, because getting a lot of features for free is great, as long
-as we can extend everything when we need to. To get the most “easy” out of 
-EasyAdmin, you should definitely code along with me. 
+Hey friends! We are in for a treat with this tutorial! It's EasyAdmin, my favorite admin generated for Symfony. It just gives you so many features out of the box and it looks great. This shouldn't really be a surprise because its creator is the always-impressive Javier Aguila. So let's have some fun and learn how to bend EasyAdmin to our will, because getting a lot of features for free is great, as long as we can extend everything when we need to. To get the most “easy” out of  EasyAdmin, you should definitely code along with me. 
```

Comment:
Super minor, but we want true `"` instead of the fancier `“` - you might need a more basic editor to avoid having those added. I really like "Atom - it's made by GitHub :).

Reply:
Lee actually helped me set up a page on VS Code that I can toss edits into and see the results as I go. The first round I did was just tossed into a Google doc, and I guess it decided only fancy ticks are appropriate. :D Seems the VS Code page defaults to the normal ones, so that hopefully won't be a problem going forward.